akka-distributed-data/src/main/scala/akka/cluster/ddata/Replicator.scala

```diff
@@ -1222,7 +1223,9 @@ final class Replicator(settings: ReplicatorSettings) extends Actor with ActorLog
       }
     } match {
       case Success((envelope, delta)) ⇒
-        log.debug("Received Update for key [{}], old data [{}], new data [{}], delta [{}]", key, localValue, envelope.data, delta)
+        if (log.isDebugEnabled)
+          log.debug("Received Update for key [{}], old data [{}], new data [{}], delta [{}]", key,
+            localValue.map(_.data.getClass.getName), envelope.data.getClass.getName, delta.map(_.getClass.getName))
```

Comment:
The classes are not very interesting here, so let's make this 
`"Received Update for key [{}]"`